models/brats_mri_generative_diffusion/configs/metadata.json

```diff
@@ -0,0 +1,62 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
```

Comment:
this schema is not compatible with `network_data_format` defined below. Could you double confirm the format?
You can also refer to:
https://github.com/Project-MONAI/model-zoo/blob/dev/models/mednist_gan/configs/metadata.json#L2

Reply:
after fixing the issue, you can try to run:
`python ci/verify_bundle.py -b brats_mri_generative_diffusion` locally to check if there are other errors.

Reply:
what is the `spatial_shape` required on the input?